plugins/ua_network_tcp.c

```diff
@@ -291,12 +291,6 @@ ServerNetworkLayerTCP_closeConnection(UA_Connection *connection) {
         return;
     connection->state = UA_CONNECTION_CLOSED;
 #endif
-#if UA_LOGLEVEL <= 300
```

Comment:
i still do not get why you deleted this? the "normal" network layer has the pointer to itself